Create sweep with ID: v1kbp485
Sweep URL: https://wandb.ai/cs23s025-indian-institute-of-technology-madras/----project-name----/sweeps/v1kbp485
Collecting sklearn
  Using cached sklearn-0.0.post12.tar.gz (2.6 kB)
  Preparing metadata (setup.py) ... [?25l- error
  [1;31merror[0m: [1msubprocess-exited-with-error[0m

  [31mÃ—[0m [32mpython setup.py egg_info[0m did not run successfully.
  [31mâ”‚[0m exit code: [1;36m1[0m
  [31mâ•°â”€>[0m [31m[15 lines of output][0m
  [31m   [0m The 'sklearn' PyPI package is deprecated, use 'scikit-learn'
  [31m   [0m rather than 'sklearn' for pip commands.
  [31m   [0m
  [31m   [0m Here is how to fix this error in the main use cases:
  [31m   [0m - use 'pip install scikit-learn' rather than 'pip install sklearn'
  [31m   [0m - replace 'sklearn' by 'scikit-learn' in your pip requirements files
  [31m   [0m   (requirements.txt, setup.py, setup.cfg, Pipfile, etc ...)
  [31m   [0m - if the 'sklearn' package is used by one of your dependencies,
  [31m   [0m   it would be great if you take some time to track which package uses
  [31m   [0m   'sklearn' instead of 'scikit-learn' and report it to their issue tracker
  [31m   [0m - as a last resort, set the environment variable
  [31m   [0m   SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True to avoid this error
  [31m   [0m
  [31m   [0m More information is available at
  [31m   [0m https://github.com/scikit-learn/sklearn-pypi-package
  [31m   [0m [31m[end of output][0m

  [1;35mnote[0m: This error originates from a subprocess, and is likely not a problem with pip.
[?25h[1;31merror[0m: [1mmetadata-generation-failed[0m

[31mÃ—[0m Encountered error while generating package metadata.
[31mâ•°â”€>[0m See above for output.

[1;35mnote[0m: This is an issue with the package mentioned above, not pip.
[1;36mhint[0m: See above for details.
Note: you may need to restart the kernel to use updated packages.
